### ISSUE TITLE
mac version osx

### ISSUE THREAD

**Issue by TheHaloDeveloper** [ThoNohT/NohBoard#210]
is there a mac vversion??

**Comment by austinav** [comment]
No, it is only for Windows. I might look at upgrading it to .Net MAUI which would make the new version compatible with macOS and other platforms, but I haven't completely vetted that yet either. Not sure if that would be accepted by @ThoNohT. What your thoughts on this suggestion/direction @ThoNohT?

**Comment by ThoNohT** [comment]
I don't think I would accept major changes to this anymore. Since the tool is not in development, and hasn't been for years. Anyone is free to fork it and continue development of course.

I don't think switching to .NET MAUI will fix all problems though. A bigger challenge than the UI will probably be the keyboard/mouse hooking code. I have been experimenting with this a bit in the past year on Linux/Windows. But it's nowhere near becoming anything resembling NohBoard, and I still don't have much time to work on it either.

**Comment by austinav** [comment]
> A bigger challenge than the UI will probably be the keyboard/mouse hooking code. I have been experimenting with this a bit in the past year on Linux/Windows

I hadn't thought as much about that, but that makes sense since you are reading scan codes. Any unexpected challenges that you give me a heads up on?
